mhctools/base_commandline_predictor.py

```diff
@@ -21,7 +21,7 @@
 from six import string_types
 from typechecks import require_string, require_integer, require_iterable_of
 from mhcnames import normalize_allele_name
-from mhcnames.parsing_helpers import AlleleParseError
```

Comment:
this is using the new `mhcnames` so my PR won't be necessary - it was a stopgap anyway. but we may want to add the current `mhcnames` version to requirements.txt and setup.py as part of this PR since we're here